tests/integration/test_integration.py

```diff
@@ -114,15 +105,17 @@ async def post_deployment_tasks(self, namespace: str) -> str:
         generated_token = vault.generate_token(ttl="5m")
         return generated_token
 
-    async def test_given_no_config_when_deploy_then_status_is_waiting(  # noqa: E501
-        self, ops_test, charm, cleanup
+    @pytest.mark.abort_on_fail
+    async def test_given_no_config_when_deploy_then_status_is_blocked(  # noqa: E501
+        self,
+        ops_test,
+        build_and_deploy,
```

Comment:
```suggestion
        self, ops_test, build_and_deploy
```

Reply:
Black doesn't like it in the same line.

Reply:
The trick is with the last comma ;) If you remove it, black will be happy.

Reply:
Done.